gptme/chat.py

```diff
@@ -12,10 +12,10 @@
 from .constants import PROMPT_USER
 from .init import init
 from .interrupt import clear_interruptible, set_interruptible
-from .llm import reply
+from .providers.llm import reply
```

Comment:
I thought we said not to create a module?

I think `gptme.providers.llm` is unnecessarily verbose, I much prefer the current `gptme.llm`.

Reply:
I like this!